src/Strat-LmtOrdrs.py

```diff
@@ -262,13 +268,14 @@ def run(self):
                                      , self.on_position_changed)
 
     def run_backtest(self):
-        self.ibhft = bt.Backtester()
-        self.ibhft.set_csv_file("ticks 10 mins - Jun 25 2014.csv")
+        self.ibhft = Backtester()
+        self.ibhft.set_csv_file("C:\\Users\\jloss\\PyCharmProjects\\IB-Trading-Models-And-Backtester\\src\\ticks 10 mins - Jun 25 2014.csv")
```

Comment:
This line should be reverted so that it works with relative path (absolute path will work only on the authors computer)